### PR TITLE
Fix(client|redux): checkout get charges endpoint with chargeId param

### PR DESCRIPTION
**Pull request by ivomota**
## Description

- This PR adds chargeId param to the getCharges endpoint on checkout
- Changes were made also on redux

## Checklist

- [x] The commit message follows our guidelines
- [x] Tests for the respective changes have been added
- [ ] The code is commented, particularly in hard-to-understand areas
- [x] The labels and/or milestones were added

## Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

**Review by rc23 — APPROVED** [review]

**Review by andrevechina — CHANGES_REQUESTED** [review]

File: packages/redux/src/checkout/actions/factories/fetchChargesFactory.ts [hunk 1]
```diff
@@ -27,14 +27,14 @@ import type {
  */
 const fetchChargesFactory =
   (getCharges: GetCharges) =>
-  (id: string, config?: Config) =>
+  (id: string, chargeId: string, config?: Config) =>
```
[the remark this answers is not in the document]
Comment (ivomota) [reply packages/redux/src/checkout/actions/factories/fetchChargesFactory.ts:31]: Done

File: packages/client/src/checkout/index.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export { default as patchCheckout } from './patchCheckout';
 export { default as patchDeliveryBundleUpgrades } from './patchDeliveryBundleUpgrades';
 export { default as patchCheckoutCompletePayment } from './patchCheckoutCompletePayment';
 export { default as patchGiftMessage } from './patchGiftMessage';
-export { default as getCharges } from './getCharges';
+export { default as getCharges } from './getCheckoutOrderCharge';
```
Comment (andrevechina) [inline packages/client/src/checkout/index.ts:17]:
```suggestion
export { default as getCheckoutOrderCharge } from './getCheckoutOrderCharge';
```

File: packages/redux/src/checkout/actions/factories/fetchChargesFactory.ts [hunk 3]
```diff
@@ -12,8 +12,9 @@ import type {
 } from '@farfetch/blackout-client/checkout/types';
 
 /**
```
Comment (andrevechina) [inline packages/redux/src/checkout/actions/factories/fetchChargesFactory.ts:14]: Should we rename this file to `fetchChargeFactory`?

File: packages/redux/src/checkout/actions/__tests__/fetchCharges.test.ts [hunk 4]
```diff
@@ -15,6 +15,7 @@ describe('fetchCharges() action creator', () => {
   const checkoutMockStore = (state = {}) =>
```
Comment (andrevechina) [inline packages/redux/src/checkout/actions/__tests__/fetchCharges.test.ts:15]: should we rename this file to `fetchCharge.test.ts`?

File: packages/client/src/checkout/index.ts (hunk 2 above)
Comment (andrevechina) [reply packages/client/src/checkout/index.ts:17]: This may include some additional changes on other files

**Review by anapim — APPROVED** [review]

File: packages/client/src/checkout/getCharges.ts [hunk 5]
```diff
@@ -1,21 +0,0 @@
-import client, { adaptError } from '../helpers/client';
```
Comment (andrevechina) [inline packages/client/src/checkout/getCharges.ts:1]: Is the `/orders/{id}/charges` endpoint decommissioned?

**Review by andrevechina — CHANGES_REQUESTED** [review]

File: packages/client/src/checkout/index.ts (hunk 2 above)
Comment (ivomota) [reply packages/client/src/checkout/index.ts:17]: You are right, I need to update the imports on redux actions

File: packages/client/src/checkout/getCharges.ts (hunk 5 above)
Comment (ivomota) [reply packages/client/src/checkout/getCharges.ts:1]: The mail goal of this task is fix this endpoint
The right path is:
`/orders/{id}/charges/{chargeId}`

File: packages/redux/src/checkout/actions/__tests__/fetchCharges.test.ts (hunk 4 above)
Comment (ivomota) [reply packages/redux/src/checkout/actions/__tests__/fetchCharges.test.ts:15]: done

**Review by andrevechina — APPROVED** [review]

**Pull request merged**